site/docs/v1/tech/admin-guide/webauthn.adoc

```diff
@@ -0,0 +1,137 @@
+
+===== Authenticator attachment
+
+There are two link:https://www.w3.org/TR/webauthn-2/#authenticator-attachment-modality[authenticator attachment modalities] in WebAuthn.
+
+ * `platform` - the authenticator is integrated with the client device
+ * `cross-platform` - the authenticator is removable from the client device and can be used with other client devices
+
+The workflow settings allow limiting the associated workflow to one attachment only or allowing either. The value is used to limit eligible authenticators to those that are best suited for the workflow. The authenticator attachment preference only impacts the WebAuthn registration ceremony. These are the recommended values for available WebAuthn workflows:
+
+ * Bootstrap - Cross-platform only. The bootstrap workflow can be used to sign in on a new device using WebAuthn. If the authenticator is integrated with the client device, it cannot be used on another.
+ * Re-authentication - Platform only. In order to ensure that the authenticator is available for repeated logins on the same device, it is best to limit authenticator selection to those integrated with the client device.
+
+:advanced_feature: WebAuthn cross-platform authenticators
+include::docs/v1/tech/shared/_advanced-edition-blurb-api.adoc[]
+
+:advanced_feature!:
+
+===== User verification
+
+There are three options for the link:https://www.w3.org/TR/webauthn-2/#enum-userVerificationRequirement[user verification requirement] defined by WebAuthn.
+
+ * `Required` - user verification is required to successfully complete the WebAuthn ceremony
+ * `Preferred` - user verification is preferred for the WebAuthn ceremony, but it will not fail if user verification is not provided
+ * `Discouraged` - user verification should not be provided for the WebAuthn ceremony
+
+The workflow settings allow selecting the preference for user verification. If the workflow allows a user to authenticate, it is _highly_ recommended to require user verification. If user authentication is not required, anyone with access to the authenticator could use it to authenticate as the user that owns the passkey.
+
+=== Application Configuration
+
+The application-level configuration options for WebAuthn serve as a way to override the availability of WebAuthn workflows that are configured at the tenant level.
+
+image::core-concepts/application-webauthn.png[Application Configuration - WebAuthn,width=1200, role=top-cropped,bottom-cropped]
+
+In order to override which workflows are enabled for the application, regardless of the tenant configuration, first enable the main toggle. Enabling the toggle in one of the subsections will enable that workflow for the application. Likewise, disabling the toggle in one of the subsections will disable that workflow for the application.
```

Comment:
So I take this to mean that if I disable the bootstrap workflow in the tenant, I can enable it at the application? If so, what settings are used?

Or is this only used to disable workflows that are enabled at the tenant?

Reply:
>  We'll probably just display the attachment and verification options for the workflow all the time.

Do you mean that these options will be available on both the tenant and application? That would make the most sense to me.

Is this work that will get done "sometime" or is it currently on the docket soon?

If the former, then I'd probably document that "It uses whatever the last settings were." Otherwise I might a subset of 190 a public issue and point the user to that.

Reply:
Will resolve in
- https://github.com/FusionAuth/fusionauth-issues/issues/1979